src/operator/tensor/elemwise_binary_scalar_op_basic.cc

```diff
@@ -143,10 +143,19 @@ MXNET_OPERATOR_REGISTER_BINARY_SCALAR(_backward_mul_scalar)
 .set_attr<FComputeEx>("FComputeEx<cpu>", BinaryScalarOp::ComputeEx<cpu, mshadow::op::mul>);
 
 MXNET_OPERATOR_REGISTER_BINARY_SCALAR(_div_scalar)
+.set_attr<FInferStorageType>("FInferStorageType", BinaryScalarStorageType)
```

Comment:
Let's add some description that div_scalar only operates on input.data if input is sparse. 
For example, if input of shape (100 x 100) has only 2 nnz, i.e. input.data = [5,6], scalar = nan, it will result output.data = [nan, nan] instead of 10000 nans. 

The following is the scipy behavior: 

```
>>> import scipy.sparse as spsp
>>> csr = spsp.rand(5,5,format='csr', density=0.5)
>>> csr
<5x5 sparse matrix of type '<type 'numpy.float64'>'
	with 12 stored elements in Compressed Sparse Row format>
>>> csr / np.nan
<5x5 sparse matrix of type '<type 'numpy.float64'>'
	with 12 stored elements in Compressed Sparse Row format>
>>> csr.todense() / np.nan
matrix([[ nan,  nan,  nan,  nan,  nan],
        [ nan,  nan,  nan,  nan,  nan],
        [ nan,  nan,  nan,  nan,  nan],
        [ nan,  nan,  nan,  nan,  nan],
        [ nan,  nan,  nan,  nan,  nan]])
```

Reply:
We should add such comment for _scalar_mul, too

Reply:
Added :)